clients/js/tsconfig.declarations.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "declaration": true,
+        "declarationMap": true,
+        "emitDeclarationOnly": true,
+        "outDir": "./dist/types",
```

Comment:
Curious, a trailing comma is not valid json right? I wonder why it doesn't throw. Can you confirm it's being used?